.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,17 @@
+
+name: CI
+on:
+  pull_request: {}
```

Comment:
You probably want build/validate at least on pull requests? I'm not sure if spec-prod handles that like you'd hope...

Reply:
From [the run on this PR](https://github.com/WICG/serial/runs/2242204351) it appears that it is doing the right thing. The YAML syntax here is a little weird, and just copied from the spec-prod example. I think the `: {}` bit is optional or maybe it's required since `push` has a parameter. I think this means that pull requests will trigger a run on any branch.